flask_celeryext/app.py

```diff
@@ -26,7 +26,11 @@ def setup_task_logger(logger=None, **kwargs):
 def create_celery_app(flask_app):
     """Create a Celery application."""
     celery = current_celery_app
-    celery.conf.update(flask_app.config)
+    if celery_version < '4.0':
```

Comment:
Hm, in theory a good idea, but in practice it doesn't support Python's newest version [PEP 440](https://www.python.org/dev/peps/pep-0440/) which is implemented in the [packaging utility library](https://github.com/pypa/packaging/blob/master/packaging/version.py) instead.

In other words distutils would not be enough to do version checks that match the community standard while `packaging` would. But I guess I wonder if we really need this right now since all this does is checking if it's lower.